app/assets/json/translations/locale-en.json

```diff
@@ -1275,6 +1275,13 @@
                 "proof_desc": "You'll need one of the following documents showing your address at Hunters View public housing.",
                 "title": "Right to Return - Hunters View (West Point)"
             },
+            "rtr_potrero": {
+                "address": "Potrero Terrace and Annex Address",
+                "address_desc": "Your current or former address at Potrero Terrace and Annex public housing",
+                "desc": "For households in which at least one person is a former or current resident of Potrero Terrace and Annex public housing. Please note, these units are not subsidized, the rent will not change if your income changes. ",
+                "proof_desc": "You'll need one of the following documents showing your address at Potrero Terrace and Annex public housing.",
+                "title": "Right to Return - Potrero Terrace and Annex"
```

Comment:
question: The figma designs use ""Right to Return - Potrero Terrace and Annex public housing", but that doesn't follow the pattern of `rtr_huntersview`. Can you confirm which title is correct?

Reply:
Bridget confirmed, but I just have to make an update to one spot

Reply:
@jimlin-sfgov I updated this - should match the design now